CHANGELOG.md

```diff
@@ -5,6 +5,33 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v0.1.6]
+
+### Fix
+
+#### xrpl
+
+Updates some fields in AccountSet and Payment related transactions to a pointer to allow 0 or "" values. For example:
+
+- DestinationTag
+- TickSize
+- Domain
+- WalletLocator
+- WalletSize
+- TransferRate
+
+Fixes some unmarshaling for `Payment`, `CheckCreate`. Added custom `unmarshalJSON` function.
+
+Adds more tests for setting some `asf` flags in `AccountSet`.
+
+## [v0.1.5]
```

Comment:
@GuillemGarciaDev I added this part for your addition with XLS-77d support. I hope it's correct.